netci.groovy

```diff
@@ -107,7 +107,7 @@ def CreateJob(script, runtime, osName, isPR, machineAffinityOverride = null, sho
                     }
 
                     // this agent has VS 15.7 on it, which is a min requirement for our repo now.
-                    machineAffinityOverride = 'Windows.10.Amd64.ClientRS3.DevEx.Open'
+                    machineAffinityOverride = 'windows.10.amd64.clientrs4.devex.15.8.open '
```

Comment:
We could also just drop Jenkins now, unless anyone would like to complain about VSTS?

Reply:
I'd be in favor of that. The rhel 7.2 job is the only one left that needs porting. Do you know what queue to use for rhel 7.2 machines?